### DISCUSSION TITLE
$1500 bounty for someone to help me fix Sparkle.

### DISCUSSION THREAD

**Discussion by DanielEdrisian** [sparkle-project/Sparkle#2646] [General]
I have spent way too much time on this. I've followed all the steps to integrate it, and somehow it still fails to update. It seems the Updater app isn't signed properly? The console logs say the Updater doesn't have the right entitlements/provision profile. I can't find a similar issue / discussion posted anywhere with that error.

There are dozens of possible issues, so I would love to pay someone with experience to help me resolve it.

Thank you!

**Comment by zorgiepoo** [comment]
I’m not sure that’s related. Sparkle should log errors in the app (usually sufficient) and in Autoupdate process if the installer started. I help folks here but not for money or that kind of level of support. Good luck if you find someone to pay. Otherwise there is not enough information here to act upon.
